rules/windows/image_load/sysmon_susp_office_dsparse_dll_load.yml

```diff
@@ -16,13 +16,13 @@ logsource:
     product: windows
 detection:
     selection:
-        Image:
-            - '*\winword.exe'
-            - '*\powerpnt.exe'
-            - '*\excel.exe'
-            - '*\outlook.exe'
-        ImageLoaded:
-            - '*\dsparse.dll*'
+        Image|endswith:
+            - '\winword.exe'
+            - '\powerpnt.exe'
+            - '\excel.exe'
+            - '\outlook.exe'
+        ImageLoaded|endswith:
```

Comment:
original logic is `contains`. I am not 100% sure that it suppose to be `endswith`. maybe it makes sense to ask the author?

Reply:
Fixed :)